compiler/src/iree/compiler/Dialect/Stream/Transforms/CloneToConsumers.cpp

```diff
@@ -177,6 +186,10 @@ struct CloneToConsumersPass
         break;
       }
     }
+    // Used to check iteration count for testing purposes.
+    if (printIterations) {
+      llvm::outs() << "iterationCount: " << iterationCount << "\n";
+    }
```

Comment:
These kind of things go under LLVM_DEBUG blocks instead - then a user can pass `-debug-only=iree-stream-clone-to-consumers` and see the output. This file does declare the DEBUG_TYPE but has no uses yet; you'd add this as:

```c++
LLVM_DEBUG(llvm::dbgs() << "[clone-to-consumers] iteration " << iterationCount << "/" << maxIterationCount << "\n");
```